### ISSUE TITLE
No "Pending translation" message for untranslated content

### ISSUE THREAD

**Issue by leeomara** [ytgov/yukon-ca#736]
Testing on UAT, untranslated content is published in English, with a header:

> The content of this page is being displayed in English because it is not available in Français.

Reported as: .. while the French version is not available, the usual message “En cours de traduction” doesn’t show up and the English page shows up instead. Could we look into this to have the “En cours de traduction” back.

e.g. https://yukon.cms-uat.yukon.ca/fr/node/25024

**Comment by w3creatives** [comment]
Go to this link:

https://yukon.ca/en/admin/config/regional/translate

Under Filter translatable strings >> Search for >> The content of this page is being 

Then under source string add this value in the input box:  En cours de traduction


![image](https://github.com/user-attachments/assets/c61b77ca-0f50-4018-b76c-26c27a01a600)

**Comment by leeomara** [comment]
This message appears to be originate in the Blinker core module. It can be found in `docroot/modules/contrib/blinker_core/blinker_core.module`

Unfortunately, it doesn't display to anonymous users, so it's not suitable as a message for site users.

**Comment by w3creatives** [comment]
PR - #791 should fix this issue.

**Comment by leeomara** [comment]
I think we're close. This seems to do the right thing for logged in users, however anonymous users get an "Access denied" error page instead.

<img width="772" alt="Screen Shot 2024-12-19 at 8 51 01 PM" src="https://github.com/user-attachments/assets/01c36b91-a552-4f11-bd59-ce9a335fc9f8" />

**Comment by w3creatives** [comment]
We checked and its working for us. Could you please check and confirm if the page status = published?

**Comment by leeomara** [comment]
You are correct. I had changed the node status testing another issue. It now shows "En cours de traduction" correctly.

**Comment by leeomara** [comment]
This is in place for production.
